frontend/src/index.js

```diff
@@ -25,11 +24,9 @@ ReactDOM.render(
             <SettingsProvider>
               <BrowserRouter>
                 <SnackbarProvider dense maxSnack={3} hideIconVariant>
-                  <RequestContextProvider>
-                    <AuthProvider>
-                      <App />
-                    </AuthProvider>
-                  </RequestContextProvider>
+                  <AuthProvider>
+                    <App />
+                  </AuthProvider>
```

Comment:
@noah-paige does this interfere with re-auth?

Reply:
Hi @dlpzx , I have tested the reAuth with Cognito Stack and External Idp . Works for both. Please test it once at your end to be sure.

Reply:
Thanks for the clarification and for the testing